.env.sepolia

```diff
@@ -1,4 +1,3 @@
-GENESIS_FILE_PATH=sepolia/genesis.json
```

Comment:
Isn't this still needed for Reth?
What about same variable inside `.env.mainnet`? It's still there.